src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  bookItems: [],
+  isLoading: true,
+};
+
+const booksSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addBook: () => {
+
+    },
+    removeBook: () => {
+    },
+  },
```

Comment:
- [ ] Kindly implement the actual reducers for `addBook` and `removeBook` here instead of leaving them blank.